t/object_methods.t

```diff
@@ -85,3 +85,6 @@ is($datetime3->minus_time_interval('1d')->is_same_as($datetime2),  1,          '
 is($datetime1->minus_time_interval(0),                             $datetime1, 'minus_time_interval(0) yields the same object');
 is($datetime2->minus_time_interval('-1d')->is_same_as($datetime3), 1,          'minus_time_interval("-1d") yields one day ahead.');
 throws_ok { $datetime3->minus_time_interval("one") } qr/Bad format/, 'minus_time_interval("one") is not a mind-reader..';
+
+is($datetime1->get_nth_day_of_week(3, 3)->day_of_month, 21, 'nth_day_of_week(3, 3) is correct');
```

Comment:
Also, as long as I am here, you will want the two arguments here to be distinct.  With `(3,3)` it would be possible for someone to switch the argument order in the code and still have passing tests even though the behavior would be wrong. 

The tests also act as a sort of runnable documentation, so we should be able to look at the test and know how the call works.